extensions/resteasy-reactive/quarkus-resteasy-reactive-links/deployment/src/main/java/io/quarkus/resteasy/reactive/links/deployment/LinksProcessor.java

```diff
@@ -85,14 +85,15 @@ AdditionalBeanBuildItem registerRestLinksProviderProducer() {
     }
 
     @BuildStep
-    @Produce(EmptyBuildItem.class)
+    @Produce(ArtifactResultBuildItem.class)
```

Comment:
I could not use the ValidationErrorBuildItem here because it was causing cycle detected issues. Using ArtifactResultBuildItem did work here.